README.md

```diff
@@ -10,35 +10,37 @@
 1. [Javascript Specific](#javascript)
 1. To Be Continued
 
+## Want to Test Solutions ?
+__[Demo and testing](demo)__
+ - `npm install` or `yarn install`
```

Comment:
Also, to be consistent with `install`, it might be worth adding how you test with yarn (`yarn test`)